core/pom.xml

```diff
@@ -63,6 +63,17 @@
             <artifactId>apiguardian-api</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.webjars</groupId>
+            <artifactId>jquery</artifactId>
+            <version>3.5.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.webjars</groupId>
+            <artifactId>webjars-locator-core</artifactId>
```

Comment:
This might conflict with other applications that use this as a production dependency.

You'll have to shade both.